docs/sources/mimir/release-notes/v2.14.md

```diff
@@ -0,0 +1,139 @@
+---
+title: "Grafana Mimir version 2.14 release notes"
+menuTitle: "V2.14 release notes"
+description: "Release notes for Grafana Mimir version 2.14"
+weight: 1100
+---
+
+# Grafana Mimir version 2.14 release notes
+
+<!-- vale Grafana.GoogleWill = NO -->
+<!-- vale Grafana.Timeless = NO -->
+<!-- Release notes are often future focused -->
+
+Grafana Labs is excited to announce version 2.14 of Grafana Mimir.
+
+The highlights that follow include the top features, enhancements, and bug fixes in this release.
+For the complete list of changes, refer to the [CHANGELOG](https://github.com/grafana/mimir/blob/main/CHANGELOG.md).
+
+## Features and enhancements
+
+The minimal supported version of Go is updated to 1.22.
+
+The streaming of the chunks from store-gateways to queriers is enabled by default.
+
+Alertmanager adds the `-alertmanager.log-parsing-label-matchers` configuration option to control logging when parsing label matchers.
+This flag is intended to be used with `-alertmanager.utf8-strict-mode-enabled` to validate UTF-8 strict mode is working as intended.
+The default value for the configuration option is `false`.
+
+Alertmanager adds the `-alertmanager.utf8-migration-logging-enabled` configuration option, that enables logging of tenant configurations,
```

Comment:
The alertmanager flags, compactor alert, and distributor changes don't seem like they're worth calling out in release notes.